tests/house_tests.py

```diff
@@ -22,9 +20,10 @@ def test_delete_house(self, generate_house):
 
 @pytest.mark.django_db
 class TestFilterHouseForm:
-    def test_filter_parent_profession_1(self):
+    def test_filter_parent_profession_1(self, new_user):
```

Comment:
Rony changed these tests get only self and the user is equal to UserFactory
take a look at Rony's PR
It's to save you trouble with the conflict resolution :)

Reply:
She needs to rebase and it will be fixed

Reply:
what Rony said :)